app/main/routes.py

```diff
@@ -587,7 +587,9 @@ def search_transferring_body(_id: uuid.UUID):
     page = int(request.args.get("page", 1))
 
     filters = {"query": query}
-    sorting_orders = build_sorting_orders_open_search(request.args)
+
+    sorting_query = build_sorting_orders_open_search(request.args)
+    sorting_orders = build_sorting_orders(request.args)
```

Comment:
`sorting_orders` isnt being used in this function anymore so I think this line can be removed